src/index.coffee

```diff
@@ -53,6 +47,11 @@ module.exports =
     if globalScope.console != nextConsole
       throw new Error('Wtf')
 
+    return {
+      consolePlus: nextConsole
+      browserTransport: createdConsole[1]
```

Comment:
Same as above – would not add this. Plus it may not be a "browser" transport every time.

Reply:
My reply above applies to this as well.